geopandas/geodataframe.py

```diff
@@ -631,6 +631,31 @@ def _dataframe_set_geometry(self, col, drop=False, inplace=False, crs=None):
                          " DataFrame to GeoDataFrame")
     return GeoDataFrame(self, geometry=col, crs=crs)
 
+
+def concat(L, axis=0):
+    """ Concatenate multiple GeoDataFrames or GeoSeries together
+
+    Currently only works for axis=0
```

Comment:
I would add some "expectations": currently concat assumes that all elements in the passed list are either GeoDataFrame objects or either GeoSeries, and the geometry name and crs for all should be the same (as this is not checked currently, just taken from the first)

Reply:
Added expectations